pandas/tests/groupby/test_apply.py

```diff
@@ -1559,3 +1559,45 @@ def test_include_groups(include_groups):
     if not include_groups:
         expected = expected[["b"]]
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("f", [max, min, sum])
+@pytest.mark.parametrize("keys", ["jim", ["jim", "joe"]])  # Single key  # Multi-key
```

Comment:
the comment might be clearer if instead of being a comment it was encoded in `ids=["single_key", "multi_key"]`